app/models/payment.rb

```diff
@@ -32,7 +32,14 @@ def key
   end
 
   def bill_description
-    record['billReasonDescription']
+    case record['billReasonDescription']
```

Comment:
We do some other type of translations using a hash. Is it possible to keep a similar pattern for that?

Translation:
https://github.com/sul-dlss/mylibrary/blob/master/config/application.rb#L41-L69

Used:
https://github.com/sul-dlss/mylibrary/blob/a4b72fe7221fe57331733a283f8f6dddb799cfee/app/helpers/application_helper.rb#L63

Reply:
Thanks for catching that. Done in https://github.com/sul-dlss/mylibrary/pull/411/commits/d3391b7630a140b8dea230e87ec3a11853688f68